src/animation/AnimationUtils.js

```diff
@@ -231,6 +234,112 @@ var AnimationUtils = {
 
 		return clip;
 
+	},
+
+	makeClipAdditive: function ( targetClip, referenceFrame, referenceClip, fps ) {
+
+		if ( referenceFrame === undefined ) referenceFrame = 0;
+		if ( referenceClip === undefined ) referenceClip = targetClip;
+		if ( fps === undefined || fps <= 0 ) fps = 30;
+
+		var numTracks = targetClip.tracks.length;
+		var referenceTime = referenceFrame / fps;
+
+		// Make each track's values relative to the values at the reference frame
+		for ( var i = 0; i < numTracks; ++ i ) {
+
+			var referenceTrack = referenceClip.tracks[ i ];
+			var referenceTrackType = referenceTrack.ValueTypeName;
+
+			// Skip this track if it's non-numeric
+			if ( referenceTrackType === 'bool' || referenceTrackType === 'string' ) continue;
+
+			// Find the track in the target clip whose name and type matches the reference track
+			var targetTrack = targetClip.tracks.find( function ( track ) {
+
+				return track.name === referenceTrack.name
+				&& track.ValueTypeName === referenceTrackType;
+
+			} );
+
+			if ( targetTrack === undefined ) continue;
+
+			var valueSize = referenceTrack.getValueSize();
+			var lastIndex = referenceTrack.times.length - 1;
+			var referenceValue;
+
+			// Find the value to subtract out of the track
+			if ( referenceTime <= referenceTrack.times[ 0 ] ) {
+
+				// Reference frame is earlier than the first keyframe, so just use the first keyframe
+				referenceValue = AnimationUtils.arraySlice( referenceTrack.values, 0, referenceTrack.valueSize );
```

Comment:
Not that is breaks anything, but while debugging I had `referenceTrack.valueSize` being undefined. I think it should be `referenceTrack.getValueSize()` or `valueSize`

Reply:
/ping @c-morten

Reply:
@mrdoob I actually fixed this along with some other issues in a new PR: https://github.com/mrdoob/three.js/pull/20021. This PR fixes the issue I reported here: https://github.com/mrdoob/three.js/issues/20025